File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main8Activity.java
package com.alejouribesanchez.class_4;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TableLayout;

public class Main8Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main8);

        TableLayout tableLayout = (TableLayout) findViewById(R.id.activity_main8);
        tableLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(Main8Activity.this, Main9Activity.class);
                startActivity(intent);
            }
        });
    }
}

File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main8Activity.kt
package com.alejouribesanchez.class_4

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TableLayout

class Main8Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main8)

        val tableLayout = findViewById(R.id.activity_main8) as TableLayout
        tableLayout.setOnClickListener {
            val intent = Intent(this@Main8Activity, Main9Activity::class.java)
            startActivity(intent)
        }
    }
}
